#  Awesome AI Meets Biology[![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
[![Website](https://img.shields.io/website?url=https%3A//droma01.github.io/DROMA)](https://webioinfo01.github.io/Awesome-AI-Meets-Biology/)
[![GitHub Stars](https://img.shields.io/github/stars/Webioinfo01/Awesome-AI-Meets-Bioinformatics_Biomedicine?style=social)](https://github.com/Webioinfo01/Awesome-AI-Meets-Biomedicine) 
[![visitors](https://visitor-badge.laobi.icu/badge?page_id=Webioinfo01.Awesome-AI-Meets-Bioinformatics_Biomedicine)](https://github.com/Webioinfo01/Awesome-AI-Meets-Bioinformatics_Biomedicine)

**Visit the official website at: https://webioinfo01.github.io/Awesome-AI-Meets-Biology/**

AI separated into "Foundation models" and "AI Agents" for biology/biomedical/bioinformatics research.

## üìã Table of Contents
- [üåü AI Agents](#AI-Agents)
- [üéØ Foundation models](#Foundation-models)
- [üõ†Ô∏è AI Tools](#AI-Tools)
- [üíæ Databases/Simulation](#DatabasesSimulation)
- [üìä Benchmarks](#Benchmarks)
- [üìö Reviews](#Reviews)
- [üîó Other Awesome Projects](#Other-Awesome-Projects)

## AI Agents

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **PromptBio: A Multi-Agent AI Platform for Bioinformatics Data Analysis** | PromptBio(Xiao Yang) | [Link](https://www.promptbio.ai/) |  | multiple agents system for bioinformatics | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.07.05.663295v1) | [Link](https://www.promptbio.ai/download.html) |
| 2025.07 | **GPTBioInsightor** |  |  |  | Single Cell annotation |  |  | [Link](https://github.com/huang-sh/GPTBioInsightor) ![GitHub Stars](https://img.shields.io/github/stars/huang-sh/GPTBioInsightor) |
| 2025.06 | **NVIDIA Biomedical AI-Q Research Agent Developer Blueprint** | NVIDIA | [Link](https://github.com/NVIDIA) |  | Drug Discovery (Target Identification) | Nvidia's blog | [Link](https://build.nvidia.com/nvidia/biomedical-aiq-research-agent) | [Link](https://github.com/NVIDIA/bionemo-framework) ![GitHub Stars](https://img.shields.io/github/stars/NVIDIA/bionemo-framework) |
| 2025.06 | **OriGene: A Self-Evolving Virtual Disease Biologist Automating Therapeutic Target Discovery** |  |  |  | General Biomedical Research (therapeutic targets) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.03.657658v1) | [Link](https://gentel-lab.github.io/OriGene-Homepage/) ![GitHub Stars](https://img.shields.io/github/stars/GENTEL-lab/OriGene) |
| 2025.06 | **Agent Laboratory: Using LLM Agents as Research Assistants** |  |  |  | General Scientific Research | arXiv | [Link](https://arxiv.org/abs/2501.04227) | [Link](https://agentlaboratory.github.io/) ![GitHub Stars](https://img.shields.io/github/stars/SamuelSchmidgall/AgentLaboratory) |
| 2025.06 | **scExtract: leveraging large language models for fully automated single-cell RNA-seq data annotation and prior-informed multi-dataset integration** |  |  |  | scRNA downstream tasks | Genome Biology | [Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03639-x) | [Link](https://github.com/yxwucq/scExtract) ![GitHub Stars](https://img.shields.io/github/stars/yxwucq/scExtract) |
| 2025.06 | **CellVoyager: AI CompBio Agent Generates New Insights by Autonomously Analyzing Biological Data** | James Zou | [Link](https://github.com/zou-group) |  | single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.03.657517v1) | [Link](https://github.com/zou-group/CellVoyager) ![GitHub Stars](https://img.shields.io/github/stars/zou-group/CellVoyager) |
| 2025.05 | **Alita: Generalist Agent Enabling Scalable Agentic Reasoning with Minimal Predefinition and Maximal Self-Evolution** | Mengdi Wang | [Link](https://mwang.princeton.edu/) |  | General Scientific Research(self-evolution and create MCP) | bioRxiv | [Link](https://arxiv.org/abs/2505.20286) | [Link](https://github.com/CharlesQ9/Alita) ![GitHub Stars](https://img.shields.io/github/stars/CharlesQ9/Alita) |
| 2025.05 | **BioOmni: A General-Purpose AI Agent for Automated Biomedical Research** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | General Biomedical Research (Multi-modal) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.05.30.656746v1) | [Link](https://biomni.stanford.edu/) ![GitHub Stars](https://img.shields.io/github/stars/snap-stanford/biomni) |
| 2025.05 | **CellTypeAgent: Trustworthy cell type annotation with Large Language Models** | Yunjian Li |  |  | LLM agent for cell type annotation in single-cell data | ArXiv | [Link](https://www.semanticscholar.org/paper/40e125e1b882b278d14d7f86617a4381084fa69a) |  |
| 2025.05 | **ChatMolData: A Multimodal Agent for Automatic Molecular Data Processing** | Xiaohui Yu |  |  | Multimodal LLM-agent for automatic molecular data processing | Advanced Intelligent Systems | [Link](https://www.semanticscholar.org/paper/1fd6efbd486e07872b17bf8b08ab68c5a60716c0) |  |
| 2025.05 | **PlantGPT: An Arabidopsis-Based Intelligent Agent that Answers Questions about Plant Functional Genomics.** | Qinlong Zhu |  |  | PlantGPT: LLM agent for plant functional genomics question answering | Advanced science | [Link](https://www.semanticscholar.org/paper/b0b726a254ede59afe4ab6635fb23c916ceacb94) |  |
| 2025.05 | **DrugPilot: LLM-based Parameterized Reasoning Agent for Drug Discovery** | Wenbin Hu |  |  | LLM-based agent for parameterized reasoning in drug discovery | ArXiv | [Link](https://www.semanticscholar.org/paper/a51b28d8fcf17077a038fe775117865731f9c98d) |  |
| 2025.05 | **Automatic biomarker discovery and enrichment with BRAD** | I. Rajapakse |  |  | LLM agent for automatic biomarker discovery and enrichment (BRAD) | Bioinformatics | [Link](https://www.semanticscholar.org/paper/3ac8cac9583281d5eebca6a17e00af015eb8b286) |  |
| 2025.04 | **The AI Scientist-v2: Workshop-Level Automated Scientific Discovery via Agentic Tree Search** | Sakana AI(David Ha) | [Link](https://github.com/SakanaAI) |  | General Scientific Research | ICLR 2025 | [Link](https://arxiv.org/abs/2504.08066) | [Link](https://github.com/SakanaAI/AI-Scientist-v2) ![GitHub Stars](https://img.shields.io/github/stars/SakanaAI/AI-Scientist-v2) |
| 2025.04 | **SpatialAgent: An autonomous AI agent for spatial biology** | Genentech(Aviv Regev) | [Link](https://github.com/Genentech) |  | Spatial scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.03.646459v1) | [Link](https://github.com/Genentech/SpatialAgent) ![GitHub Stars](https://img.shields.io/github/stars/Genentech/SpatialAgent) |
| 2025.04 | **Knowledge Graph and Large Language Model for Metabolomics** | Yuxing Lu |  |  | Knowledge Graphs and LLMs in Metabolomics |  | [Link](https://www.semanticscholar.org/paper/3d7a503eded6fabd6979c3847d5db68970bf9d3d) |  |
| 2025.04 | **Spatial transcriptomics AI agent charts hPSC-pancreas maturation in vivo** | Jia Liu |  | Harvard University | AI Agent for Spatial Transcriptomics Analysis | bioRxiv | [Link](https://www.semanticscholar.org/paper/1c45bfa74d72bc131f7254ae28e5d7ee1abd425f) |  |
| 2025.04 | **scAgent: Universal Single-Cell Annotation via a LLM Agent** | Yunjun Gao |  |  | Universal Single-Cell Annotation with LLM Agent | ArXiv | [Link](https://www.semanticscholar.org/paper/0e8d9a99ce3f609b18e039f9770ef935e114b02e) |  |
| 2025.04 | **Large Language Model Consensus Substantially Improves the Cell Type Annotation Accuracy for scRNA-seq Data** |  |  |  | Single Cell annotation | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.10.647852v1) | [Link](https://github.com/cafferychen777/mLLMCelltype) ![GitHub Stars](https://img.shields.io/github/stars/cafferychen777/mLLMCelltype) |
| 2025.04 | **SCassist: An AI Based Workflow Assistant for Single-Cell Analysis** | Rachel R Caspi |  | National Institutes of Health | Single Cell tasks(cluster, annotation) | Bioinformatics | [Link](https://www.biorxiv.org/content/10.1101/2025.04.22.650107v1) | [Link](https://github.com/NIH-NEI/SCassist) ![GitHub Stars](https://img.shields.io/github/stars/NIH-NEI/SCassist) |
| 2025.04 | **Fleming: An AI Agent for Antibiotic Discovery in Mycobacterium tuberculosis** | M. Farhat |  | Harvard Medical School | Tuberculosis antibiotic discovery AI agent | bioRxiv | [Link](https://www.semanticscholar.org/paper/d4b49553e887d04229c6cd04cc1eb6e464332e4e) |  |
| 2025.04 | **OLAF: An Open Life Science Analysis Framework for Conversational Bioinformatics Powered by Large Language Models** | Diego Gonzalez Lopez |  |  | Conversational bioinformatics pipeline | ArXiv | [Link](https://www.semanticscholar.org/paper/4305e3ff95f996eafc5a25dec044eed4af34aac4) |  |
| 2025.03 | **TxAgent: An AI Agent for Therapeutic Reasoning Across a Universe of Tools** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | Personalized Treatment Recommendations | arXiv | [Link](https://arxiv.org/abs/2503.10970) | [Link](https://github.com/mims-harvard/TxAgent) ![GitHub Stars](https://img.shields.io/github/stars/mims-harvard/TxAgent) |
| 2025.03 | **DrBioRight 2.0: an LLM-powered bioinformatics chatbot for large-scale cancer functional proteomics analysis** |  |  |  | proteomics | Nature Communication | [Link](https://www.nature.com/articles/s41467-025-57430-4) | [Link](https://drbioright.org/) |
| 2025.03 | **DrugAgent: Automating AI-aided Drug Discovery Programming through LLM Multi-Agent Collaboration** |  |  |  | Drug Discovery | arXiv | [Link](https://arxiv.org/abs/2411.15692) | [Link](https://anonymous.4open.science/r/drugagent-5C42/) |
| 2025.03 | **PharmAgents: Building a Virtual Pharma with Large Language Model Agents** | Yanyan Lan |  |  | Drug Discovery | ArXiv | [Link](https://www.semanticscholar.org/paper/46e921cb6fba3423e48fa894afa9b0b47bf934d6) |  |
| 2025.03 | **AstroAgents: A Multi-Agent AI for Hypothesis Generation from Mass Spectrometry Data** | Amirali Aghazadeh |  |  | Astrobiology & Mass Spectrometry AI | ArXiv | [Link](https://www.semanticscholar.org/paper/a967904a87ba4c1d6359fbd7342a074d63e2fe63) |  |
| 2025.03 | **DORA AI Scientist: Multi-agent Virtual Research Team for Scientific Exploration Discovery and Automated Report Generation** | Alex Zhavoronkov |  | Insilico Medicine | Scientific Research Automation | bioRxiv | [Link](https://www.semanticscholar.org/paper/b913baa47245a4a76054bb12e04d61c3c88fb532) |  |
| 2025.03 | **Design and Analysis of an Extreme-Scale, High-Performance, and Modular Agent-Based Simulation Platform** | Lukas Breitwieser |  |  | Agent-Based Simulation | ArXiv | [Link](https://www.semanticscholar.org/paper/ab1f61ae171dfbe425e60bc942997c1bdba29c24) |  |
| 2025.03 | **Collaborative Expert LLMs Guided Multi-Objective Molecular Optimization** | Haishuai Wang |  |  | Molecular Optimization | ArXiv | [Link](https://www.semanticscholar.org/paper/391ca60f6cd9b7ccc0ff8bc5fc72c6543c770fd9) |  |
| 2025.03 | **IAN: An Intelligent System for Omics Data Analysis and Discovery** | Rachel R. Caspi |  | Laboratory of Immunology, National Eye Institute, NIH, Bethesda 20892, USA | Omics Data Integration | bioRxiv | [Link](https://www.semanticscholar.org/paper/1ecbcfe40fe475b6fec3186ced39162afdb33175) | [Link](https://github.com/NIH-NEI/IAN) ![GitHub Stars](https://img.shields.io/github/stars/NIH-NEI/IAN) |
| 2025.02 | **LIDDIA: Language-based Intelligent Drug Discovery Agent** |  |  |  | Drug Discovery | arXiv | [Link](https://arxiv.org/abs/2502.13959) |  |
| 2025.02 | **Towards an AI co-scientist** |  |  |  | General Scientific Research | arXiv | [Link](https://arxiv.org/pdf/2502.18864v1) |  |
| 2025.02 | **Knowledge Synthesis of Photosynthesis Research Using a Large Language Model** | Tae In Ahn |  |  | Photosynthesis research assistant using LLM | ArXiv | [Link](https://www.semanticscholar.org/paper/cb217acb0ea521307477a39ec893f135b72ead45) |  |
| 2025.02 | **Spike sorting AI agent** | Jia Liu |  | John A. Paulson School of Engineering and Applied Sciences, Harvard University, Boston, MA, USA | Spike sorting AI pipeline agent | bioRxiv | [Link](https://www.semanticscholar.org/paper/e44999184842c94e6f976417cbe73431501eac03) |  |
| 2025.02 | **RAPID: Reliable and efficient Automatic generation of submission rePorting checklists with Large language moDels** | Lu Zhang |  | Hong Kong Baptist University | Automated medical reporting checklist generation | bioRxiv | [Link](https://www.semanticscholar.org/paper/8ab71d93085aaa5900ad6ec5e7b966829934d95e) |  |
| 2025.01 | **BioMaster: Multi-agent System for Automated Bioinformatics Analysis Workflow** |  |  |  | Multi-omics Pipelines | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.01.23.634608v1) | [Link](https://github.com/ai4nucleome/BioMaster) ![GitHub Stars](https://img.shields.io/github/stars/ai4nucleome/BioMaster) |
| 2025.01 | **InstructCell: A Multi-Modal AI Copilot for Single-Cell Analysis with Instruction Following** |  |  |  | single cell RNA | arXiv | [Link](https://arxiv.org/abs/2501.08187) | [Link](https://github.com/zjunlp/InstructCell) ![GitHub Stars](https://img.shields.io/github/stars/zjunlp/InstructCell) |
| 2025.01 | **BioAgents: Democratizing Bioinformatics Analysis with Multi-Agent Systems** | Microsoft Research(Venkat S. Malladi) | [Link](https://github.com/microsoft) |  | Multi-omics Pipelines | arXiv | [Link](https://arxiv.org/abs/2501.06314) |  |
| 2025.01 | **Extracting Knowledge from Scientific Texts on Patient-Derived Cancer Models Using Large Language Models: Algorithm Development and Validation** | G. Savova |  | Boston Children's Hospital, Harvard Medical School | LLM-based entity extraction for patient-derived cancer models | bioRxiv | [Link](https://www.semanticscholar.org/paper/4ea31174978b289183898b4cafd1432d9ddd0349) |  |
| 2024.12 | **CASSIA: a multi-agent large language model for reference free, interpretable, and automated cell annotation of single-cell RNA-sequencing data** |  |  |  | Single Cell annotation | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.12.04.626476v2) | [Link](https://github.com/ElliotXie/CASSIA) ![GitHub Stars](https://img.shields.io/github/stars/ElliotXie/CASSIA) |
| 2024.12 | **ProtChat: An AI Multi-Agent for Automated Protein Analysis Leveraging GPT-4 and Protein Language Model** | Yunpeng Cai |  |  | Automated protein analysis AI agent | Journal of chemical information and modeling | [Link](https://www.semanticscholar.org/paper/c89cfdd40cf24c40b09304cbbf85024933def10c) |  |
| 2024.12 | **SCREADER: Prompting Large Language Models to Interpret scRNA-seq Data** | Meng Xiao |  |  | scRNA-seq interpretation via LLM prompting | 2024 IEEE International Conference on Data Mining Workshops (ICDMW) | [Link](https://www.semanticscholar.org/paper/aba63c2f387349a58bbc9a89d83e2018bc436819) |  |
| 2024.12 | **SciAgents: Automating Scientific Discovery Through Bioinspired Multi‚ÄêAgent Intelligent Graph Reasoning** | Markus J. Buehler |  |  | Multi-agent AI for automated materials discovery | Advanced Materials (Deerfield Beach, Fla.) | [Link](https://www.semanticscholar.org/paper/dbbcdb281ed6aa646af0402172cf8a7cfda85d5c) |  |
| 2024.12 | **Development and Application of an In Vitro Drug Screening Assay for Schistosoma mansoni Schistosomula Using YOLOv5** | Antonio Muro |  |  | AI-powered drug screening assay for schistosomula | Biomedicines | [Link](https://www.semanticscholar.org/paper/0cac159098bb2365ca8547979f6f81018165a0ca) |  |
| 2024.11 | **BioDiscoveryAgent: An AI Agent for Designing Genetic Perturbation Experiments** |  |  |  | Experimental Design (Genetic Perturbation) | arXiv | [Link](https://arxiv.org/abs/2405.17631) | [Link](https://github.com/snap-stanford/BioDiscoveryAgent/) ![GitHub Stars](https://img.shields.io/github/stars/snap-stanford/BioDiscoveryAgent) |
| 2024.11 | **The Virtual Lab: AI Agents Design New SARS-CoV-2 Nanobodies with Experimental Validation** | James Zou | [Link](https://github.com/zou-group) |  | General Scientific Research | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.11.11.623004v1) | [Link](https://github.com/zou-group/virtual-lab) ![GitHub Stars](https://img.shields.io/github/stars/zou-group/virtual-lab) |
| 2024.10 | **An AI Agent for Fully Automated Multi-Omic Analyses** |  |  |  | Multi-omics Pipelines | Advanced Science | [Link](https://advanced.onlinelibrary.wiley.com/doi/10.1002/advs.202407094) | [Link](https://github.com/JoshuaChou2018/AutoBA) ![GitHub Stars](https://img.shields.io/github/stars/JoshuaChou2018/AutoBA) |
| 2024.09 | **The AI Scientist: Towards Fully Automated Open-Ended Scientific Discovery** | Sakana AI(David Ha) | [Link](https://github.com/SakanaAI) |  | General Scientific Research | arXiv | [Link](https://arxiv.org/abs/2408.06292) | [Link](https://github.com/SakanaAI/AI-Scientist) ![GitHub Stars](https://img.shields.io/github/stars/SakanaAI/AI-Scientist) |
| 2024.07 | **CellAgent: An LLM-driven Multi-Agent Framework for Automated Single-cell Data Analysis** |  |  |  | single cell RNA | arXiv | [Link](https://arxiv.org/abs/2407.09811) | [Link](http://cell.agent4science.cn/) |
| 2024.05 | **A Data-Intelligence-Intensive Bioinformatics Copilot System for Large-scale Omics Researches and Scientific Insights** |  |  |  | Multi-omics/single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.05.19.594895v2) | [Link](https://github.com/lyyang01/Bio-Copilot) ![GitHub Stars](https://img.shields.io/github/stars/lyyang01/Bio-Copilot) |
| 2024.04 | **CRISPR-GPT: An LLM Agent for Automated Design of Gene-Editing Experiments** | Mengdi Wang, Le Cong | [Link](https://mwang.princeton.edu/) |  | Experimental Design (CRISPR Genetic Editing) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.04.25.591003v3) | [Link](https://github.com/cong-lab/crispr-gpt-pub) ![GitHub Stars](https://img.shields.io/github/stars/cong-lab/crispr-gpt-pub) |
| 2024.01 | **ProtAgents: Protein discovery via large language model multi-agent collaborations combining physics and machine learning** |  |  |  | Protein design and discovery | arXiv | [Link](https://arxiv.org/abs/2402.04268) | [Link](https://github.com/lamm-mit/ProtAgents) ![GitHub Stars](https://img.shields.io/github/stars/lamm-mit/ProtAgents) |

## Foundation models

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **ODFormer: a Virtual Organoid for Predicting Personalized Therapeutic Responses in Pancreatic Cancer** |  |  |  | Organoid drug response prediction | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.07.08.663664v2) | [Link](https://github.com/microsoft/bioemu) ![GitHub Stars](https://img.shields.io/github/stars/xujing363/ODFormer) |
| 2025.07 | **Scalable emulation of protein equilibrium ensembles with generative deep learning** | Microsoft Research(Frank No√©) | [Link](https://github.com/microsoft) |  | Protein | Science | [Link](https://www.science.org/doi/10.1126/science.adv9817) | [Link](https://github.com/microsoft/bioemu) ![GitHub Stars](https://img.shields.io/github/stars/microsoft/bioemu) |
| 2025.07 | **Spatia: Multimodal Model for Prediction and Generation of Spatial Cell Phenotypes** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | spatial single cell RNA | ArXiv | [Link](https://arxiv.org/html/2507.04704v1#S4) |  |
| 2025.07 | **RiNALMo: general-purpose RNA language models can generalize well on structure prediction tasks** |  |  |  | RNA | Nature Communication | [Link](https://www.nature.com/articles/s41467-025-60872-5) | [Link](https://github.com/lbcb-sci/RiNALMo) ![GitHub Stars](https://img.shields.io/github/stars/lbcb-sci/RiNALMo) |
| 2025.06 | **World Models as Simulators of Patient Biology: Oncology Counterfactual Therapeutics Oracle (OCTO)** | Noetik | [Link](https://www.noetik.ai/research) |  | spatial single cell RNA perturbation | noetik's report | [Link](https://www.noetik.ai/octo) |  |
| 2025.06 | **Generalized biological foundation model with unified nucleic acid and protein language** | Alibaba(Zhaorong Li) | [Link](https://github.com/LucaOne) |  | DNA/RNA/Protein | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-025-01044-4) | [Link](https://github.com/LucaOne/LucaOne) ![GitHub Stars](https://img.shields.io/github/stars/LucaOne/LucaOne) |
| 2025.06 | **Predicting cellular responses to perturbation across diverse contexts with State** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) |  | genetic, signaling, and chemical perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661135v1) | [Link](https://github.com/ArcInstitute/state) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/state) |
| 2025.06 | **AlphaGenome: Advancing regulatory variant effect prediction with a unified DNA sequence model** | Google DeepMind(Pushmeet Kohli) | [Link](https://github.com/google-deepmind) |  | DNA/RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.25.661532v1) | [Link](https://github.com/google-deepmind/alphagenome) ![GitHub Stars](https://img.shields.io/github/stars/google-deepmind/alphagenome) |
| 2025.06 | **SPACE: Your Genomic Profile Predictor is a Powerful DNA Foundation Model** |  |  |  | DNA | ICML 2025 | [Link](https://arxiv.org/abs/2506.01833) | [Link](https://github.com/YangLab/CSI) ![GitHub Stars](https://img.shields.io/github/stars/YangLab/CSI) |
| 2025.06 | **UniCure: A Foundation Model for Predicting Personalized Cancer Therapy Response** |  |  |  | drug perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.14.658531v1) | [Link](https://github.com/ZexiChen502/UniCure) ![GitHub Stars](https://img.shields.io/github/stars/ZexiChen502/UniCure) |
| 2025.06 | **A multimodal conversational agent for DNA, RNA and protein tasks** | InstaDeep(Thomas Pierrot) | [Link](https://huggingface.co/InstaDeepAI) |  | Diverse Omics include DNA/RNA/Protein | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-025-01047-1) | [Link](https://huggingface.co/InstaDeepAI/ChatNT) |
| 2025.05 | **A visual‚Äìomics foundation model to bridge histopathology with spatial transcriptomics** | Guangyu Wang | [Link](https://guangyuwanglab.github.io/web/) |  | histopathology and spatial single cell RNA | Nature Methods | [Link](https://www.nature.com/articles/s41592-025-02707-1) | [Link](https://github.com/GuangyuWangLab2021/Loki) ![GitHub Stars](https://img.shields.io/github/stars/GuangyuWangLab2021/Loki) |
| 2025.05 | **GeneBreaker: Jailbreak Attacks against DNA Language Models with Pathogenicity Guidance** | Mengdi Wang | [Link](https://mwang.princeton.edu/) |  | Biosafety for DNA | arXiv | [Link](https://arxiv.org/abs/2505.23839) | [Link](https://github.com/zaixizhang/GeneBreaker) ![GitHub Stars](https://img.shields.io/github/stars/zaixizhang/GeneBreaker) |
| 2025.05 | **BioReason: Incentivizing Multimodal Biological Reasoning within a DNA-LLM Model** | Bo Wang | [Link](https://github.com/bowang-lab) |  | DNA | arXiv | [Link](https://arxiv.org/abs/2505.23579) | [Link](https://github.com/bowang-lab/BioReason) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/BioReason) |
| 2025.05 | **CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells** |  |  |  | single cell RNA | Nature Communications | [Link](https://www.nature.com/articles/s41467-025-59926-5) | [Link](https://github.com/biomed-AI/CellFM) ![GitHub Stars](https://img.shields.io/github/stars/biomed-AI/CellFM) |
| 2025.05 | **sciLaMA: A Single-Cell Representation Learning Framework to Leverage Prior Knowledge from Large Language Models** | G. Quon |  | University of California, Davis | Single-cell representation learning leveraging LLM prior knowledge | bioRxiv | [Link](https://www.semanticscholar.org/paper/9aab32b1ffef407d9918bcf6f7b87c930d45c3f7) |  |
| 2025.05 | **LlamaAffinity: A Predictive Antibody‚ÄìAntigen Binding Model Integrating Antibody Sequences with Llama3 Backbone Architecture** | J. Chen |  | University of Alabama at Birmingham | Antibody‚Äìantigen binding affinity prediction using LLM-based models | bioRxiv | [Link](https://www.semanticscholar.org/paper/2478b5379054be98bb2b0f325bc474ff806f1ea6) |  |
| 2025.05 | **GRAPE: Heterogeneous Graph Representation Learning for Genetic Perturbation with Coding and Non-Coding Biotype** | Stan Z. Li |  |  | Graph representation learning for genetic perturbation integrating LLM and DNA features | ArXiv | [Link](https://www.semanticscholar.org/paper/e8cd781a1d5b928a0ff4d5eb2c3d42fdefdd6239) |  |
| 2025.04 | **Scaling Large Language Models for Next-Generation Single-Cell Analysis** | David van Dijk | [Link](https://github.com/vandijklab) |  | single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.14.648850v1) | [Link](https://github.com/vandijklab/cell2sentence) ![GitHub Stars](https://img.shields.io/github/stars/vandijklab/cell2sentence) |
| 2025.04 | **CellFlow enables generative single-cell phenotype modeling with flow matching** | Fabian J. Theis | [Link](https://github.com/theislab) |  | Perturbation single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.11.648220v1) | [Link](https://github.com/theislab/CellFlow) ![GitHub Stars](https://img.shields.io/github/stars/theislab/CellFlow) |
| 2025.04 | **Abstract 5084: Evaluation of single-cell foundation models for cancer outcome predictions** | Eliezer M Van Allen |  |  | Single-cell Foundation Models for Cancer Outcome Prediction | Cancer Research | [Link](https://www.semanticscholar.org/paper/e8db25fc84442d27305675bc67f101ee2cad8d49) |  |
| 2025.04 | **Abstract 6316: Predictive performance comparison of foundational and CNN models for single-cell immune profiling** | Mai Chan Lau |  |  | Foundational vs. CNN Models for Immune Profiling in Histology | Cancer Research | [Link](https://www.semanticscholar.org/paper/bf2a8036a813c5bc51269f833d5c0109c7275950) |  |
| 2025.04 | **Abstract 5059: Self-supervised representation learning of somatic mutational data** | Etai Jacob |  |  | Self-supervised Representation Learning for Somatic Mutation Data | Cancer Research | [Link](https://www.semanticscholar.org/paper/f4b0d1ee35796530e33a20b8115b704361e4c180) |  |
| 2025.04 | **dGeneralized Biological Foundation Model with Unified Nucleic Acid and Protein Language** | Zhaorong Li |  | Alibaba Cloud Intelligence, Alibaba Group, Hangzhou, China | Unified DNA/RNA/Protein foundation model | bioRxiv | [Link](https://www.semanticscholar.org/paper/3f784482894a47708c6584f667ad38de70a3ba76) |  |
| 2025.04 | **Towards multimodal foundation models in molecular cell biology.** | Bo Wang |  |  | Multimodal molecular biology foundation models | Nature | [Link](https://www.semanticscholar.org/paper/7f65931fb6bbfca0496c919e83ed40e910b2dc5f) |  |
| 2025.04 | **Abstract 3762: DEL-AI: Proteome-wide in silico screening of multi-billion compound libraries using machine learning foundation models** | Paul Novick |  |  | Proteome-wide in silico DEL screening | Cancer Research | [Link](https://www.semanticscholar.org/paper/555b53a4b3f89030f96e6a2e2d7e4ee83a1de4b5) |  |
| 2025.03 | **Multimodal AI predicts clinical outcomes of drug combinations from preclinical data** | Marinka Zitnik |  |  | Clinical Pharmacology Modeling | ArXiv | [Link](https://www.semanticscholar.org/paper/342b7a61048fb4025a64078633f530fd3f74dbc7) |  |
| 2025.02 | **Genome modeling and design across all domains of life with Evo 2** | Arc Institute(Brian L. Hie) | [Link](https://github.com/ArcInstitute) |  | DNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.18.638918v1) | [Link](https://github.com/arcinstitute/evo2) ![GitHub Stars](https://img.shields.io/github/stars/arcinstitute/evo2) |
| 2025.02 | **GENERator: A Long-Context Generative Genomic Foundation Model** |  |  |  | DNA | arXiv | [Link](https://arxiv.org/abs/2502.07272) | [Link](https://github.com/GenerTeam/GENERator) ![GitHub Stars](https://img.shields.io/github/stars/GenerTeam/GENERator) |
| 2025.02 | **scGPT-spatial: Continual Pretraining of Single-Cell Foundation Model for Spatial Transcriptomics** | Bo Wang | [Link](https://github.com/bowang-lab) |  | spatial single cell RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.05.636714v1) | [Link](https://github.com/bowang-lab/scGPT-spatial) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/scGPT-spatial) |
| 2025.02 | **Large Cognition Model: Towards Pretrained EEG Foundation Model** | Aidan Hung-Wen Tsai |  |  | EEG foundation model | ArXiv | [Link](https://www.semanticscholar.org/paper/92e6303157e9044ada38869c8d3c318882bbb9b6) |  |
| 2025.02 | **Omni-DNA: A Unified Genomic Foundation Model for Cross-Modal and Multi-Task Learning** | Caihua Shan |  |  | Cross-modal genomic foundation model | ArXiv | [Link](https://www.semanticscholar.org/paper/7b9c7b43df7282797547f622788389d09cf5b3d2) |  |
| 2025.02 | **GENERator: A Long-Context Generative Genomic Foundation Model** | Zheng Wang |  |  | Generative genomic foundation model | ArXiv | [Link](https://www.semanticscholar.org/paper/4f98b1f62cfd71e2d2bf44156a5a971029cf4c16) |  |
| 2025.02 | **Top-DTI: Integrating Topological Deep Learning and Large Language Models for Drug Target Interaction Prediction** | S. Bozdag |  | University of North Texas | Drug‚Äìtarget interaction prediction framework | bioRxiv | [Link](https://www.semanticscholar.org/paper/c7aa64927a5600588c7e6600aafcf1a3b0d3e41b) |  |
| 2025.02 | **AI-enabled alkaline-resistant evolution of protein to apply in mass production** | Liang Hong |  |  | LLM-driven protein evolution for alkaline resistance | eLife | [Link](https://www.semanticscholar.org/paper/91b7e83815ad6a4da3232dd62f3799634ee43dda) |  |
| 2025.01 | **A foundation model of transcription across human cell types** |  |  |  | transcriptional regulation(ATAC-seq) | Nature | [Link](https://www.nature.com/articles/s41586-024-08391-z) | [Link](https://github.com/GET-Foundation) ![GitHub Stars](https://img.shields.io/github/stars/GET-Foundation/get_model) |
| 2025.01 | **GENA-LM: a new DNA language model for long sequences** |  |  |  | DNA | Nucleic Acids Research | [Link](https://academic.oup.com/nar/article/53/2/gkae1310/7954523) | [Link](https://github.com/AIRI-Institute/GENA_LM) ![GitHub Stars](https://img.shields.io/github/stars/AIRI-Institute/GENA_LM) |
| 2025.01 | **Function-Guided Conditional Generation Using Protein Language Models with Adapters** | Salesforce Research( Ali Madani) | [Link](https://github.com/salesforce) |  | Protein | arXiv | [Link](https://arxiv.org/abs/2410.03634) | [Link](https://github.com/Profluent-Internships/ProCALM) ![GitHub Stars](https://img.shields.io/github/stars/Profluent-Internships/ProCALM) |
| 2025.01 | **Simulating 500 million years of evolution with a language model** | EvolutionaryScale(Alexander Rives) | [Link](https://github.com/evolutionaryscale) |  | Protein | Science | [Link](https://www.science.org/doi/10.1126/science.ads0018) | [Link](https://github.com/evolutionaryscale/esm) ![GitHub Stars](https://img.shields.io/github/stars/evolutionaryscale/esm) |
| 2025.01 | **Predicting cell morphological responses to perturbations using generative modeling** | Fabian J. Theis, Mohammad Lotfollahi | [Link](https://github.com/theislab) |  | phenotype morphological responses to perturbation | Nature Communications | [Link](https://www.nature.com/articles/s41467-024-55707-8) | [Link](https://github.com/theislab/IMPA) ![GitHub Stars](https://img.shields.io/github/stars/theislab/IMPA) |
| 2025.01 | **Improving functional protein generation via foundation model-derived latent space likelihood optimization** | C√©sar de la Fuente-Nunez |  | University of Pennsylvania | Generative protein design via PLM latent space optimization | bioRxiv | [Link](https://www.semanticscholar.org/paper/98be9134fce5ba8889ea6de518b814c9482ecd8e) |  |
| 2025.01 | **Unveiling the Evolution of Antimicrobial Peptides in Gut Microbes via Foundation Model-Powered Framework** | Jinfang Zheng |  | Zhejiang Lab | Antimicrobial peptide discovery from gut microbes | bioRxiv | [Link](https://www.semanticscholar.org/paper/2fc9811c567186a15873eb8600df36b27f40c088) |  |
| 2025.01 | **Knowledge Hierarchy Guided Biological-Medical Dataset Distillation for Domain LLM Training** | Meng Xiao |  |  | Biomedical dataset distillation for LLM training | ArXiv | [Link](https://www.semanticscholar.org/paper/8d6ab0e57aa1f7d20e680f3306a9e098a9b3a286) |  |
| 2024.12 | **L2G: Repurposing Language Models for Genomics Tasks** | Ameet Talwalkar |  | Carnegie Mellon University | Repurposing LLMs for genomics tasks | bioRxiv | [Link](https://www.semanticscholar.org/paper/238d842ee1679545e00d212a342f271a05473eaf) |  |
| 2024.12 | **Fine-Tuned Deep Transfer Learning Models for Large Screenings of Safer Drugs Targeting Class A GPCRs** | M. Filizola |  | Icahn School of Medicine at Mount Sinai | Deep transfer learning for GPCR drug screening | bioRxiv | [Link](https://www.semanticscholar.org/paper/adfd39e5060b00b74a38d7b7243f60af2934e966) |  |
| 2024.12 | **Porter 6: Protein Secondary Structure Prediction by Leveraging Pre-Trained Language Models (PLMs)** | G. Pollastri |  |  | Protein secondary structure prediction with PLMs | International Journal of Molecular Sciences | [Link](https://www.semanticscholar.org/paper/4eb9efc6ea3f7bf7935c374940af4c3866a5a368) |  |
| 2024.12 | **ProtDAT: A Unified Framework for Protein Sequence Design from Any Protein Text Description** | Hong-Bin Shen |  |  | Multi-modal protein sequence design from text | ArXiv | [Link](https://www.semanticscholar.org/paper/1fc064d589849dfefce0d28fe64b9b83600a0cb6) |  |
| 2024.11 | **Nucleotide Transformer: building and evaluating robust foundation models for human genomics** |  |  |  | DNA | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02523-z) | [Link](https://github.com/instadeepai/nucleotide-transformer) ![GitHub Stars](https://img.shields.io/github/stars/instadeepai/nucleotide-transformer) |
| 2024.10 | **Orthrus: Towards Evolutionary and Functional RNA Foundation Models** |  |  |  | RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.10.10.617658v1) | [Link](https://github.com/bowang-lab/Orthrus) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/Orthrus) |
| 2024.09 | **stFormer: a foundation model for spatial transcriptomics** |  |  |  | DNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.09.27.615337v6) | [Link](https://github.com/csh3/stFormer) ![GitHub Stars](https://img.shields.io/github/stars/csh3/stFormer) |
| 2024.08 | **BioRAG: A RAG-LLM Framework for Biological Question Reasoning** |  |  |  | Life science question answer (RAG) | arXiv | [Link](https://arxiv.org/abs/2408.01107) |  |
| 2024.07 | **DNA language model GROVER learns sequence context in the human genome** |  |  |  | DNA | Nature Machine Intelligence | [Link](https://www.nature.com/articles/s42256-024-00872-0) | [Link](https://zenodo.org/records/13374192) |
| 2024.06 | **scFoundation: Large-scale foundation model on single-cell transcriptomics** | Biomap research(Le Song, Xuegong Zhang) | [Link](https://github.com/biomap-research) |  | single cell RNA | Nature Communication | [Link](https://www.nature.com/articles/s41592-024-02305-7) | [Link](https://github.com/biomap-research/scFoundation) ![GitHub Stars](https://img.shields.io/github/stars/biomap-research/scFoundation) |
| 2024.06 | **Improving Medical Reasoning through Retrieval and Self-Reflection with Retrieval-Augmented Large Language Models** |  |  |  | biomedical question answer (RAG) | ISMB 24 | [Link](https://arxiv.org/abs/2401.15269) | [Link](https://github.com/dmis-lab/self-biorag) ![GitHub Stars](https://img.shields.io/github/stars/dmis-lab/self-biorag) |
| 2024.06 | **Multi-modal Transfer Learning between Biological Foundation Models** | InstaDeep(Guillaume Richard, Thomas Pierrot) | [Link](https://huggingface.co/InstaDeepAI) |  | Diverse Omics include DNA/RNA/Protein | arXiv | [Link](https://arxiv.org/abs/2406.14150) | [Link](https://huggingface.co/InstaDeepAI/isoformer) |
| 2024.05 | **Accurate structure prediction of biomolecular interactions with AlphaFold 3** | Google DeepMind(John M. Jumper) | [Link](https://github.com/google-deepmind) |  | Protein | Nature | [Link](https://www.nature.com/articles/s41586-024-07487-w) | [Link](https://github.com/google-deepmind/alphafold3) ![GitHub Stars](https://img.shields.io/github/stars/google-deepmind/alphafold3) |
| 2024.04 | **Masked Autoencoders for Microscopy are Scalable Learners of Cellular Biology** | Recursion(Berton Earnshaw) | [Link](https://www.rxrx.ai/) |  | microscopy data | CVPR 2024 Highlight | [Link](https://arxiv.org/abs/2404.10242) | [Link](https://www.rxrx.ai/phenom) |
| 2024.03 | **Caduceus: Bi-Directional Equivariant Long-Range DNA Sequence Modeling** |  |  |  | DNA | arXiv | [Link](https://arxiv.org/abs/2403.03234) | [Link](https://caduceus-dna.github.io/) ![GitHub Stars](https://img.shields.io/github/stars/kuleshov-group/caduceus) |
| 2024.03 | **Nicheformer: a foundation model for single-cell and spatial omics** | Fabian J. Theis | [Link](https://github.com/theislab) |  | single-cell and spatial omics | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.04.15.589472v1) | [Link](https://github.com/theislab/nicheformer) ![GitHub Stars](https://img.shields.io/github/stars/theislab/nicheformer) |
| 2024.02 | **scGPT: toward building a foundation model for single-cell multi-omics using generative AI** | Bo Wang | [Link](https://github.com/bowang-lab) |  | single cell RNA | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02201-0) | [Link](https://github.com/bowang-lab/scGPT) ![GitHub Stars](https://img.shields.io/github/stars/bowang-lab/scGPT) |
| 2024.01 | **OmniNA: A foundation model for nucleotide sequences** |  |  |  | DNA/RNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.01.14.575543v1) |  |
| 2024.01 | **xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein** | Biomap research(Le Song) | [Link](https://github.com/biomap-research) |  | Protein | arXiv | [Link](https://arxiv.org/abs/2401.06199) | [Link](https://github.com/biomap-research/xTrimoPGLM) ![GitHub Stars](https://img.shields.io/github/stars/biomap-research/xTrimoPGLM) |
| 2024.01 | **Geneverse: A Collection of Open-source Multimodal Large Language Models for Genomic and Proteomic Research** |  |  |  | DNA/Protein | EMNLP 2024 | [Link](https://openreview.net/forum?id=g1NplcWH7X) | [Link](https://github.com/HelloWorldLTY/Geneverse) ![GitHub Stars](https://img.shields.io/github/stars/HelloWorldLTY/Geneverse) |
| 2023.11 | **HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution** |  |  |  | DNA | NeurIPS 2023 | [Link](https://papers.neurips.cc/paper_files/paper/2023/file/86ab6927ee4ae9bde4247793c46797c7-Paper-Conference.pdf) | [Link](https://github.com/HazyResearch/hyena-dna) ![GitHub Stars](https://img.shields.io/github/stars/HazyResearch/hyena-dna) |
| 2023.10 | **ProGen2: Exploring the boundaries of protein language models** | Salesforce Research( Ali Madani) | [Link](https://github.com/salesforce) |  | Protein | Cell Systems | [Link](https://pubmed.ncbi.nlm.nih.gov/37909046/) | [Link](https://github.com/salesforce/progen) ![GitHub Stars](https://img.shields.io/github/stars/salesforce/progen) |
| 2023.06 | **DNABERT-2: Efficient Foundation Model and Benchmark For Multi-Species Genome** |  |  |  | DNA | ICLR 2024 | [Link](https://arxiv.org/abs/2306.15006) | [Link](https://github.com/MAGICS-LAB/DNABERT_2) ![GitHub Stars](https://img.shields.io/github/stars/MAGICS-LAB/DNABERT_2) |
| 2023.05 | **Transfer learning enables predictions in network biology** |  |  |  | General scRNA tasks, discovery of key network regulators and candidate therapeutic targets | Nature | [Link](https://www.nature.com/articles/s41586-023-06139-9) | [Link](https://huggingface.co/ctheodoris/Geneformer) |
| 2021.07 | **DNABERT: pre-trained Bidirectional Encoder Representations from Transformers for DNA-language** |  |  |  | DNA | Bioinformatics | [Link](https://academic.oup.com/bioinformatics/article/37/15/2112/6128680) | [Link](https://github.com/jerryji1993/DNABERT) ![GitHub Stars](https://img.shields.io/github/stars/jerryji1993/DNABERT) |

## AI Tools

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **GPTBioInsightor** |  |  |  | Single Cell annotation |  |  | [Link](https://github.com/huang-sh/GPTBioInsightor) ![GitHub Stars](https://img.shields.io/github/stars/huang-sh/GPTBioInsightor) |
| 2025.06 | **scExtract: leveraging large language models for fully automated single-cell RNA-seq data annotation and prior-informed multi-dataset integration** |  |  |  | scRNA downstream tasks | Genome Biology | [Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03639-x) | [Link](https://github.com/yxwucq/scExtract) ![GitHub Stars](https://img.shields.io/github/stars/yxwucq/scExtract) |
| 2025.04 | **Large Language Model Consensus Substantially Improves the Cell Type Annotation Accuracy for scRNA-seq Data** |  |  |  | Single Cell annotation | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.10.647852v1) | [Link](https://github.com/cafferychen777/mLLMCelltype) ![GitHub Stars](https://img.shields.io/github/stars/cafferychen777/mLLMCelltype) |
| 2025.04 | **SCassist: An AI Based Workflow Assistant for Single-Cell Analysis** |  |  |  | Single Cell tasks(cluster, annotation) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.04.22.650107v1) | [Link](https://github.com/NIH-NEI/SCassist) ![GitHub Stars](https://img.shields.io/github/stars/NIH-NEI/SCassist) |
| 2024.12 | **CASSIA: a multi-agent large language model for reference free, interpretable, and automated cell annotation of single-cell RNA-sequencing data** |  |  |  | Single Cell annotation | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.12.04.626476v2) | [Link](https://github.com/ElliotXie/CASSIA) ![GitHub Stars](https://img.shields.io/github/stars/ElliotXie/CASSIA) |

## Databases/Simulation

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.08 | **Drug Response Omics association MAp (DROMA, ÂçìÁéõ)** |  |  |  | Biomarker discovery, Drug prediction,(18 projects with 2,600+ samples and 56,000+ drugs, with full ecosystem) |  |  | [Link](https://github.com/mugpeng/DROMA) ![GitHub Stars](https://img.shields.io/github/stars/mugpeng/DROMA) |
| 2025.06 | **X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models** |  |  |  | Genome-wide Perturb-seq Datasets | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.11.659105v1) | [Link](https://doi.org/10.25452/figshare.plus.29190726) |
| 2025.05 | **scCompass: An Integrated Multi‚ÄêSpecies scRNA‚Äêseq Database for AI‚ÄêReady** | Yuanchun Zhou |  |  | Multi-species scRNA-seq database for AI-ready applications | Advanced Science | [Link](https://www.semanticscholar.org/paper/f5af8e3f7acd968b7baeb788f5b33cf7d868616e) |  |
| 2025.05 | **AlphaLasso‚Äîa web server to identify loop and lasso motifs in 3D structure of biopolymers** | Joanna I. Sulkowska |  |  | Web server for lasso motifs in biopolymer 3D structures (AlphaLasso) | Nucleic Acids Research | [Link](https://www.semanticscholar.org/paper/b2f16f79e6dc815d09b2892db7d3ca1d8a766478) |  |
| 2025.04 | **scMultiSim: simulation of single-cell multi-omics and spatial data guided by gene regulatory networks and cell‚Äìcell interactions** |  |  |  | Simulation of single-cell multi-omics and spatial data | Nature Methods | [Link](https://www.nature.com/articles/s41592-025-02651-0) | [Link](https://github.com/ZhangLabGT/scMultiSim/) ![GitHub Stars](https://img.shields.io/github/stars/ZhangLabGT/scMultiSim) |
| 2025.04 | **uHAF: a unified hierarchical annotation framework for cell type standardization and harmonization** | Xuegong Zhang |  |  | Hierarchical Framework for Cell Type Annotation and Harmonization | Bioinformatics | [Link](https://www.semanticscholar.org/paper/e3ce9b990c9604648ad39e38275cd61b7711415a) |  |
| 2025.04 | **OmniCellTOSG: The First Cell Text-Omic Signaling Graphs Dataset for Joint LLM and GNN Modeling** | Fuhai Li |  |  | Text-omic signaling graph dataset | ArXiv | [Link](https://www.semanticscholar.org/paper/fd87bf03430232aa9c2aeb733bd8d6dc023c4ef7) |  |
| 2025.04 | **Abstract 1087: The evolving landscape of cancer transcriptomics data** | Akp√©li V. Nordor |  |  | Cancer transcriptomics data mapping | Cancer Research | [Link](https://www.semanticscholar.org/paper/fcfbebdd995f3f1fc4592e08c4ed7e08c5ae2da1) |  |
| 2025.03 | **RxRx3-core: Benchmarking drug-target interactions in High-Content Microscopy** | Recursion(Imran S. Haque) | [Link](https://www.rxrx.ai/) |  | microscopy perturbation data | ICLR 2025 | [Link](https://arxiv.org/abs/2503.20158v2) | [Link](https://www.rxrx.ai/datasets) |
| 2025.02 | **scBaseCount: an AI agent-curated, uniformly processed, and continually expanding single cell data repository** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) |  | preprocess scRNA data | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.27.640494v2) | [Link](https://github.com/ArcInstitute/SRAgent) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/SRAgent) |
| 2025.02 | **Tahoe-100M: A Giga-Scale Single-Cell Perturbation Atlas for Context-Dependent Gene Function and Cellular Modeling** | tahoebio(but in arc's repository) | [Link](https://github.com/ArcInstitute) |  | Drug perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.02.20.639398v3) | [Link](https://github.com/ArcInstitute/arc-virtual-cell-atlas) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/arc-virtual-cell-atlas) |
| 2025.02 | **Literature-scaled immunological gene set annotation using AI-powered immune cell knowledge graph (ICKG)** | Ken Chen |  | MD Anderson Cancer Center | Immune cell knowledge graph for gene set annotation | bioRxiv | [Link](https://www.semanticscholar.org/paper/b51970fcc636348932f171cf220d810b5a873ba0) |  |
| 2024.12 | **M3-20M: A Large-Scale Multi-Modal Molecule Dataset for AI-driven Drug Design and Discovery** | Shuigeng Zhou |  |  | Multi-modal molecule dataset for drug design | Journal of bioinformatics and computational biology | [Link](https://www.semanticscholar.org/paper/aa3a1bbd4f6ec6934ce6a3850991747f563ab3b0) |  |
| 2024.12 | **BioMedGraphica: An All-in-One Platform for Biomedical Prior Knowledge and Omic Signaling Graph Generation** | Fuhai Li |  | Washington University in St. Louis | Biomedical knowledge graph platform | bioRxiv | [Link](https://www.semanticscholar.org/paper/212898d49afdf60a12eeda9935bc54d748da3089) |  |
| 2024.12 | **Basic Science and Pathogenesis.** | Li-San Wang |  |  | AI-enhanced search for Alzheimer's genomic database | Alzheimer's & dementia : the journal of the Alzheimer's Association | [Link](https://www.semanticscholar.org/paper/28d30ecee02ba1039bd83c77d3131c7c744dad26) |  |
| 2024.06 | **Signals in the Cells: Multimodal and Contextualized Machine Learning Foundations for Therapeutics** | Marinka Zitnik | [Link](https://zitniklab.hms.harvard.edu/) |  | Drug Discovery (Therapeutic science, 66 datasets) | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2024.06.12.598655v3) | [Link](https://tdcommons.ai/) ![GitHub Stars](https://img.shields.io/github/stars/mims-harvard/TDC) |
| 2024.04 | **Three million images and morphological profiles of cells treated with matched chemical and genetic perturbations (Cell Painting Gallery)** | Broad Institute(Anne E. Carpenter) | [Link](https://github.com/broadinstitute) |  | phenotype morphological responses to perturbation | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02241-6) | [Link](https://github.com/broadinstitute/cellpainting-gallery/) ![GitHub Stars](https://img.shields.io/github/stars/broadinstitute/cellpainting-gallery) |
| 2024.01 | **scPerturb: harmonized single-cell perturbation data** |  |  |  | scRNA perturbation(44 public datasets, drugs and genes) | Nature Methods | [Link](https://www.nature.com/articles/s41592-023-02144-y) | [Link](https://github.com/sanderlab/scPerturb/) ![GitHub Stars](https://img.shields.io/github/stars/sanderlab/scPerturb) |

## Benchmarks

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.06 | **Fundamental Limitations of Foundation Models in Single-Cell Transcriptomics** |  |  |  | evaluate cell-type classification | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661767v1) |  |
| 2025.06 | **Cell-Eval(Predicting cellular responses to perturbation across diverse contexts with State)** | Arc Institute(Yusuf H. Roohani) | [Link](https://github.com/ArcInstitute) |  | Perturbation scRNA | bioRxiv | [Link](https://www.biorxiv.org/content/10.1101/2025.06.26.661135v1) | [Link](https://github.com/ArcInstitute/cell-eval) ![GitHub Stars](https://img.shields.io/github/stars/ArcInstitute/cell-eval) |
| 2025.05 | **The influence of prompt engineering on large language models for protein‚Äìprotein interaction identification in biomedical literature** | Yi-Hsuan Lin |  |  | Prompt engineering of LLMs for protein-protein interaction extraction | Scientific Reports | [Link](https://www.semanticscholar.org/paper/8738c78f5a2c2d94dcfc34ada047ed7e51dfbe8a) |  |
| 2025.05 | **CellVerse: Do Large Language Models Really Understand Cell Biology?** | P. Heng |  |  | LLMs for language-driven single-cell multi-omics analysis (CellVerse) | ArXiv | [Link](https://www.semanticscholar.org/paper/4dd1c628a26c7b13c98e975c7ae8b5c16fbc320e) |  |
| 2025.05 | **Large Language Models Can Extract Metadata for Annotation of Human Neuroimaging Publications** | Jessica A Turner |  | Ohio State University Wexner Medical Center, Columbus, OH, USA | LLM extraction and annotation of neuroimaging metadata | bioRxiv | [Link](https://www.semanticscholar.org/paper/59087a91539883877289ceb562272f1cc13e9545) |  |
| 2025.05 | **scDrugMap: Benchmarking Large Foundation Models for Drug Response Prediction** | Qianqian Song |  |  | Benchmarking foundation models for single-cell drug response prediction | ArXiv | [Link](https://www.semanticscholar.org/paper/2dde8c82b5b59d46224f68e760ce5965a847e9c2) |  |
| 2025.04 | **Zero-shot evaluation reveals limitations of single-cell foundation models** | Microsoft Research(Alex X. Lu) | [Link](https://github.com/microsoft) |  | cell type clustering and batch integration for scGPT and Geneformer | Genome Biology | [Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-025-03574-x) | [Link](https://github.com/microsoft/zero-shot-scfoundation) ![GitHub Stars](https://img.shields.io/github/stars/microsoft/zero-shot-scfoundation) |
| 2025.04 | **OmniCellTOSG: The First Cell Text-Omic Signaling Graphs Dataset for Joint LLM and GNN Modeling** | Fuhai Li |  |  | Cell Text-Omic Signaling Graph Benchmark | ArXiv | [Link](https://www.semanticscholar.org/paper/fd87bf03430232aa9c2aeb733bd8d6dc023c4ef7) |  |
| 2025.04 | **Abstract 6316: Predictive performance comparison of foundational and CNN models for single-cell immune profiling** | Mai Chan Lau |  |  | Single-cell immune profiling benchmark | Cancer Research | [Link](https://www.semanticscholar.org/paper/bf2a8036a813c5bc51269f833d5c0109c7275950) |  |
| 2025.03 | **Distinct social-linguistic processing between humans and large audio-language models: Evidence from model-brain alignment** | Zhenguang G. Cai |  |  | Cognitive Neuroscience & Audio-Language Modeling | ArXiv | [Link](https://www.semanticscholar.org/paper/47e39f3d5f62dbf4079be48a77f540baea3f689d) |  |
| 2025.02 | **BioMaze: Benchmarking and Enhancing Large Language Models for Biological Pathway Reasoning** | Zhi-Hong Deng |  |  | Biological pathway reasoning benchmark | ArXiv | [Link](https://www.semanticscholar.org/paper/1456f9a8717ea2987897172a3e855970302cc373) |  |
| 2025.02 | **The Skin Game: Revolutionizing Standards for AI Dermatology Model Comparison** | Dariusz Jemielniak |  |  | Dermatology image classification evaluation framework | ArXiv | [Link](https://www.semanticscholar.org/paper/79dd24bdeaefadf1681d1d68104a02ba78c9c91d) |  |
| 2025.02 | **Consequences of training data composition for deep learning models in single-cell biology** | Lorin Crawford |  | Harvard Medical School | Training data composition effects in single-cell models | bioRxiv | [Link](https://www.semanticscholar.org/paper/fa6f62ba24bfa8f771c0f01f6a0ecbd81b253088) |  |
| 2025.01 | **CARDBiomedBench: A Benchmark for Evaluating Large Language Model Performance in Biomedical Research** | F. Faghri |  | Center for Alzheimer's and Related Dementias, National Institute on Aging, National Institutes of Health, Bethesda, MD, 20892, USA; DataTecnica, Washington, | LLM evaluation in neurodegenerative disease research | bioRxiv | [Link](https://www.semanticscholar.org/paper/8b9312c65220e7ef9c5edfc5b3b9f8bb259e1444) |  |
| 2025.01 | **Large Language Models Think Too Fast To Explore Effectively** | Robert C. Wilson |  |  | Exploration capabilities of LLMs in open-ended tasks | ArXiv | [Link](https://www.semanticscholar.org/paper/7d4202a3e3801d6304e0e2463c49d033b37da77d) |  |
| 2025.01 | **Sequence Modeling Is Not Evolutionary Reasoning** | M. Zitnik |  | Harvard Medical School | Evolutionary reasoning benchmark for protein LLMs | bioRxiv | [Link](https://www.semanticscholar.org/paper/f62b34cfa37bc9066affe185da5349f4c9ef9fb0) |  |
| 2024.12 | **PerturBench: Benchmarking Machine Learning Models for Cellular Perturbation Analysis** |  |  |  | Perturbation scRNA | NeurIPS 2024 | [Link](https://neurips.cc/virtual/2024/102911) | [Link](https://github.com/altoslabs/perturbench) ![GitHub Stars](https://img.shields.io/github/stars/altoslabs/perturbench) |
| 2024.12 | **Does your model understand genes? A benchmark of gene properties for biological and text models** | Y. Shimoni |  |  | Benchmark of gene property prediction models | ArXiv | [Link](https://www.semanticscholar.org/paper/7eb6e6bce59f3e724ae3365b1348627dfd73c409) |  |

## Reviews

| Year | Title | Team | Team Website | Affiliation | Domain | Venue | Paper/ Source | Code/Product |
| -----| ------| -----| -------------| ------------| -------| ------| --------------| -------------|
| 2025.07 | **Human interpretable grammar encodes multicellular systems biology models to democratize virtual cell laboratories** | Paul Macklin |  | Indiana University | Virtual cell | Cell | [Link](https://pubmed.ncbi.nlm.nih.gov/40713951/) | [Link](https://github.com/physicell-models/grammar_samples) ![GitHub Stars](https://img.shields.io/github/stars/physicell-models/grammar_samples) |
| 2025.07 | **The generative era of medical AI** | Pranav Rajpurkar |  | Harvard Medical School | Medical AI | Cell | [Link](https://pubmed.ncbi.nlm.nih.gov/40645169/) |  |
| 2025.05 | **Empowering Biomedical Research with Foundation Models in Computational Microscopy: A Systematic Review** | Rong Luo |  |  | Foundation models in computational microscopy | Advanced Intelligent Systems | [Link](https://www.semanticscholar.org/paper/c1a1785a284ae67ac3fcdfa4fc096a589762e3e2) |  |
| 2025.04 | **Model Context Protocol (MCP): Landscape, Security Threats, and Future Research Directions** |  |  |  | MCP for agents | arXiv | [Link](https://arxiv.org/abs/2503.23278) |  |
| 2025.04 | **Towards multimodal foundation models in molecular cell biology.** | Bo Wang |  |  | Multimodal Foundation Models in Molecular Cell Biology | Nature | [Link](https://www.semanticscholar.org/paper/7f65931fb6bbfca0496c919e83ed40e910b2dc5f) |  |
| 2025.04 | **A Survey of the Model Context Protocol (MCP): Standardizing Context to Enhance Large Language Models (LLMs)** |  |  |  | MCP for agents | Preprints | [Link](https://www.preprints.org/manuscript/202504.0245/v1) |  |
| 2025.04 | **Decoding cancer prognosis with deep learning: the ASD-cancer framework for tumor microenvironment analysis** | J. Haran |  |  | Deep Learning for Tumor Microenvironment Prognosis | mSystems | [Link](https://www.semanticscholar.org/paper/6336c71a68d9435fef00a61cc1930909ac92136f) |  |
| 2025.03 | **Large language model for knowledge synthesis and AI-enhanced biomanufacturing.** | Yinjie J. Tang |  |  | Biomanufacturing | Trends in biotechnology | [Link](https://www.semanticscholar.org/paper/12610a7b39efe0ec0a1cd52c242a712ea719fb4c) |  |
| 2025.03 | **Biological Sequence with Language Model Prompting: A Survey** | Yu Li |  |  | Sequence Analysis | ArXiv | [Link](https://www.semanticscholar.org/paper/9ad1cc1a90c204bc066861e41c7d3c4125ad6340) |  |
| 2025.03 | **AI-Empowered Genome Decoding: Applications of Large Language Models in Genomics** | Yu Zhou |  |  | Genomics | Frontiers of Digital Education | [Link](https://www.semanticscholar.org/paper/fb2f674c38dfd2fb692d08588fa1f99758bce28b) |  |
| 2025.03 | **Large Language Models in Bioinformatics: A Survey** | Yu Li |  |  | Bioinformatics | ArXiv | [Link](https://www.semanticscholar.org/paper/aef0d2c0359d97c688f89c13732ec21a351ead8d) |  |
| 2025.03 | **A Conceptual Framework for Human-AI Collaborative Genome Annotation** | Radoslaw Suchecki |  |  | Genome Annotation | ArXiv | [Link](https://www.semanticscholar.org/paper/b5bbf10cc21c28b6a7b626f76869aa8f30506f78) |  |
| 2025.02 | **Triple Phase Transitions: Understanding the Learning Dynamics of Large Language Models from a Neuroscience Perspective** | Yu Takagi |  |  | LLM learning dynamics from a neuroscience perspective | ArXiv | [Link](https://www.semanticscholar.org/paper/49d8af23298ab111494ddc9a79ae6a9d89181217) |  |
| 2025.02 | **Position: Episodic Memory is the Missing Piece for Long-Term LLM Agents** | Mariya Toneva |  |  | Episodic memory framework for long-term LLM agents | ArXiv | [Link](https://www.semanticscholar.org/paper/f85004321e43370535088042ebac96b958880c5d) |  |
| 2025.02 | **Microbial Ecology to Ocean Carbon Cycling: From Genomes to Numerical Models** | E. Zakem |  |  | Integration of microbial ecology and numerical models for ocean carbon cycling | Annual Review of Earth and Planetary Sciences | [Link](https://www.semanticscholar.org/paper/dc5335e5ec2bf80abda630ada4eb431a173ddec4) |  |
| 2025.01 | **Survey and Improvement Strategies for Gene Prioritization with Large Language Models** | Xia Hu |  |  | LLM-based gene prioritization | ArXiv | [Link](https://www.semanticscholar.org/paper/7ca21dec02c418f010d6b95de3bde7bb6ac69c66) |  |
| 2025.01 | **Computational Protein Science in the Era of Large Language Models (LLMs)** | Qing Li |  |  | Computational protein science with LLMs | ArXiv | [Link](https://www.semanticscholar.org/paper/0322d6eef1567b8b6ce3998d03f955082f9a87b6) |  |
| 2025.01 | **Foundation models in bioinformatics** | Jianxin Wang |  |  | Bioinformatics foundation models overview | National Science Review | [Link](https://www.semanticscholar.org/paper/3f21f2fadc60b7f627b5154a9944ec22d28c2678) |  |
| 2025.01 | **AI Methods for Antimicrobial Peptides: Progress and Challenges** | C√©sar de la Fuente-Nunez |  |  | AI methods for antimicrobial peptide design | Microbial Biotechnology | [Link](https://www.semanticscholar.org/paper/9c310642e1a07abeab7c6a8216ec60328feb74a2) |  |
| 2025.01 | **Artificial intelligence driven innovations in biochemistry: A review of emerging research frontiers** | M. A. Lateef Junaid |  |  | AI innovations in biochemistry | Biomolecules and Biomedicine | [Link](https://www.semanticscholar.org/paper/fa06241dfa98d42974532dd314b87d14ad5dc33f) |  |
| 2025.01 | **Artificial Intelligence Tools Addressing Challenges of Cancer Progression Due to Antimicrobial Resistance in Pathogenic Biofilm Systems** | Abhijit G. Banerjee |  |  | AI tools for antimicrobial resistance in cancer biofilms | Artificial Intelligence Evolution | [Link](https://www.semanticscholar.org/paper/a86fd4458cbbe7f1ea491d93c1b5ec31da8d272d) |  |
| 2025.01 | **Learning the language of life with AI.** | E. Topol |  |  | Multiomic foundation models for biomolecule prediction | Science | [Link](https://www.semanticscholar.org/paper/864031c9cc2195153f6fc08fed9d67131ddee8c4) |  |
| 2024.12 | **Large language models facilitating modern molecular biology and novel drug development** | Fei Liu |  |  | Review of LLMs in molecular biology and drug development | Frontiers in Pharmacology | [Link](https://www.semanticscholar.org/paper/d7157f73a59c3a619549217e6d27884bebedf419) |  |
| 2024.12 | **From multi-omics to predictive biomarker: AI in tumor microenvironment** | Yingli Sun |  |  | Review of AI in tumor microenvironment multi-omics | Frontiers in Immunology | [Link](https://www.semanticscholar.org/paper/6ae86bc78b187c0ca15b977156b8a2bd41d91be9) |  |
| 2024.12 | **Advancements and Applications of Protein Structure Prediction Algorithms** | Ye Chen |  |  | Review of protein structure prediction methods | Theoretical and Natural Science | [Link](https://www.semanticscholar.org/paper/6567678e2b6869ead0953704efdb926294daeebb) |  |
| 2024.10 | **Empowering biomedical discovery with AI agents** |  |  |  | Biomedical Agents | Cell | [Link](https://pubmed.ncbi.nlm.nih.gov/39486399/) |  |
| 2024.08 | **Transformers in single-cell omics: a review and new perspectives** |  |  |  | Single cell foundation models | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02353-z) |  |
| 2024.08 | **Language models for biological research: a primer** |  |  |  | biological foundation models | Nature Methods | [Link](https://www.nature.com/articles/s41592-024-02354-y) |  |
| 2024.07 | **Foundation models for bioinformatics** |  |  |  | Bioinformatics foundation models | Quantitative Biology | [Link](https://onlinelibrary.wiley.com/doi/10.1002/qub2.69) |  |

## Other Awesome Projects

| Title                                           | Project                                                                                                                                                                             |
| ----------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **Awesome-LLMs-meet-genomes**                   | [Link](https://github.com/ychuest/Awesome-LLMs-meet-genomes) ![GitHub Stars](https://img.shields.io/github/stars/ychuest/Awesome-LLMs-meet-genomes)                                 |
| **Awesome-Virtual-Cell**                        | [Link](https://github.com/Boom5426/Awesome-Virtual-Cell) ![GitHub Stars](https://img.shields.io/github/stars/Boom5426/Awesome-Virtual-Cell)                                         |
| **Awesome-Phenotypic-Drug-Discovery**           | [Link](https://github.com/Boom5426/Awesome-Phenotypic-Drug-Discovery) ![GitHub Stars](https://img.shields.io/github/stars/Boom5426/Awesome-Phenotypic-Drug-Discovery)               |
| **Awesome-LLM-Scientific-Discovery**            | [Link](https://github.com/HKUST-KnowComp/Awesome-LLM-Scientific-Discovery) ![GitHub Stars](https://img.shields.io/github/stars/HKUST-KnowComp/Awesome-LLM-Scientific-Discovery)     |
| **Awesome bioagent papers**                     | [Link](https://github.com/aristoteleo/awesome-bioagent-papers) ![GitHub Stars](https://img.shields.io/github/stars/aristoteleo/awesome-bioagent-papers)                             |
| **Awesome LLM Agents for Scientific Discovery** | [Link](https://github.com/zhoujieli/Awesome-LLM-Agents-Scientific-Discovery) ![GitHub Stars](https://img.shields.io/github/stars/zhoujieli/Awesome-LLM-Agents-Scientific-Discovery) |
| **Awesome Papers on Agents for Science**        | [Link](https://github.com/OSU-NLP-Group/awesome-agents4science) ![GitHub Stars](https://img.shields.io/github/stars/OSU-NLP-Group/awesome-agents4science)                           |
| **awesome-single-cell**                         | [Link](https://github.com/seandavi/awesome-single-cell) ![GitHub Stars](https://img.shields.io/github/stars/seandavi/awesome-single-cell)                                           |
| **Awesome-Bioinformatics**                      | [Link](https://github.com/danielecook/Awesome-Bioinformatics) ![GitHub Stars](https://img.shields.io/github/stars/danielecook/Awesome-Bioinformatics)                               |
| **awesome**                                     | [Link](https://github.com/sindresorhus/awesome) ![GitHub Stars](https://img.shields.io/github/stars/sindresorhus/awesome)                                                           |

## Others

# Ref